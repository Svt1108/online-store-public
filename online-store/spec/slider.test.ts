import Slider from "../src/slider/slider";

describe("Method drawSlider", () => {
  const testSlider = new Slider();

  test("should be defined", () => {
    expect(testSlider.drawSlider).toBeDefined;
  });

  test("renders correctly with empty data", () => {
    testSlider.drawSlider([]);
    expect(document.body.innerHTML).toMatchSnapshot();
  });
});
