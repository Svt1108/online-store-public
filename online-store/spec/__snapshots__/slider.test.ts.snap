// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Method drawSlider renders correctly 1`] = `""`;

exports[`Method drawSlider renders correctly with empty data 1`] = `""`;
